CONTRIBUTING.rst

```diff
@@ -1,6 +1,11 @@
 Contribution guidelines
 =======================
 
+This guide describes rules for how to get your contributions into Anaconda. However, if you are seeking
+for help on how to implement changes to Anaconda please follow our
```

Comment:
s/for help/help

Sorry; I was unclear when I suggested that last time.

That's the only change I'd suggest, so I'll go ahead and approve this.

Reply:
Fixed.